### ISSUE TITLE
Suggestion: Presets in the Configurator.

### ISSUE THREAD

**Issue by opensourcedcode** [ThatFruedDued/blooket-utility#79]
In the configurator, there are very limited amounts of cheats that you can use. It is a pain to find any cheats that even work now, (especially with basil removing their blooket multitool) and even harder to find them in the supported format of .json. I was thinking that it would be easier, for all of us, if there was just a few preset cheats.

**Comment by opensourcedcode** [comment]
Or, could it be possible to add the support of .js files? Instead of having to go through the pain of making and modifying the .json, you could just enter the link of the .js file and the configuration page would run that script once.

**Comment by ThatFruedDued** [comment]
I understand the appeal of having pre-selected injectables/scripts in the utility, but this project attempts to be as unopionionated as possible while still providing a useful user experience, and I believe pre-selecting injectables goes against this. I agree that there isn't enough variety of injectables out there, and this is most likely due to the fact that I am the only one developing them for my utility (and nobody else is). My original idea for the updated design is one where many different developers could work together using my easy-to-work-with API, but it seems as though developers would rather face the struggle of running code directly on Blooket than use my utility. This lack of developers could also be due to a lack of any documentation on my part, but I have not seen **any** attention regarding developing for this, even if I created documentation.

TL;DR: Probably won't happen, but I agree that the scarcity of injectables is a problem that needs fixing.

**Comment by opensourcedcode** [comment]
Yes, I know. Just for people who want to make their own, like me for example, who want to test out snippets of code and don't want to go through the whole process of doing the .json.

**Comment by opensourcedcode** [comment]
This is an incredible sandboxed website and I feel adding in more supported file types would only help it.

**Comment by Snowflake-Coder-H2o** [comment]
@opensourcedcode @ThatFruedDued So will this be a thing?

**Comment by ThatFruedDued** [comment]
Probably

**Comment by opensourcedcode** [comment]
> @opensourcedcode @ThatFruedDued So will this be a thing?

The one about the .js support will most likely. It doesn't sound like they want preset cheats though.
